io.github.sil-car.SqueezeVid.yaml

```
@@ -0,0 +1,59 @@
+# https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html#flatpak-manifest
+app-id: io.github.sil-car.SqueezeVid
```

It should be as per the documentation.
```suggestion
app-id: io.github.sil_car.SqueezeVid
```